packages/sync/src/utils/manageSnapshotTable.ts

```diff
@@ -79,3 +78,23 @@ export const dropAllSnapshotTables = async (database: TupaiaDatabase) => {
     CREATE SCHEMA ${SCHEMA};
   `);
 };
+
+export const insertSnapshotRecords = async (
+  database: TupaiaDatabase,
+  sessionId: string,
+  records: object[],
+) => {
+  const sanitizedRecords = records
+    .map(r => snakeKeys(r))
+    .map(r => ({ ...r, data: JSON.stringify(r.data) }));
+  await database.createMany(sessionId, sanitizedRecords, SCHEMA, { shouldGenerateId: false });
```

Comment:
Another bool. I think this one is even more fine, but a possible different mechanism would be the TupaiaDatabase checking whether the record type it's creating for has an id column

Reply:
Yeah I did consider that, this one is a bit tough:
- The table it's trying to insert is a snapshot table which is auto generated and there's no record_type
- There is an id column here, but it is a sequence, so that's why we want to leave it for the postgres sequence to do its job and not generate the id at application level

Kind thought this is the only way :/